src/olympia/constants/activity.py

```diff
@@ -466,6 +466,7 @@ class EDIT_RATING(_LOG):
     action_class = 'review'
     format = _('{rating} for {addon} updated.')
     show_user_to_developer = True
+    store_ip = True
```

Comment:
oh! that seems an unrelated change - were we not storing the ip for rating edits before?  Is there something to backfill?

Reply:
we were not. It's not possible to backfill this: we don't have the data